tailwind.config.ts

```diff
@@ -1,64 +1,64 @@
-import { Config } from 'tailwindcss';
```

Comment:
i can see what you meant earlier in chat. I use single quote for the finished part, should update the eslint config then.

Reply:
I faced some errors I could not resolve from lint-staged when changing to ts. unexpected token }. and expected "from" on line 3 and some random errors irrelevant to the ts files. not sure why.

Reply:
Can you tell me more about the error your get?

Reply:
![image](https://github.com/wiktoriavh/ask-good-questions/assets/73415630/cedacd84-d4c4-4ef7-a5d9-48df1f6e537c)